security.md

```diff
@@ -119,14 +119,84 @@ Add cert paths to your gemspec
 
 -------
 
-### Include checksum of released gems in your repository
-
-    require 'digest/sha2'
-    built_gem_path = 'pkg/gemname-version.gem'
-    checksum = Digest::SHA512.new.hexdigest(File.read(built_gem_path))
-    checksum_path = 'checksum/gemname-version.gem.sha512'
-    File.open(checksum_path, 'w' ) {|f| f.write(checksum) }
-    # add and commit 'checksum_path'
+### Include SHA-256 and SHA-512 checksums of released gems in your repository
+
+Checksums can be created when you are ready to release a gem. 
+
+Currently the rake task only creates an SHA-256 checksum. Run:
+
+    rake build:checksum
+
+The checksum will be placed in the `checksums/` directory.  If you track the
+checksums in your source repository, others will be able to verify the 
+authenticity of a release.
+
+Alternatively, if you'd like a script that will create both SHA-256 and SHA-512
+checksums you might use something like the following:
+
+```ruby
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "digest/sha2"
+
+VERSION_REGEX = /\d+\.\d+\.\d+([-.].+)*/.freeze
```

Comment:
This is incorrect when applied to a filename since it will include `.gem` in the version number, making everything a pre-release.

Reply:
Oooh, that's why it fails... interesting.  I will see if I can find what rubygems uses for parsing gem names.

Reply:
I knew it included the .gem, but it hadn't ever mattered until I had to deal with prereleases.